selector.go

```diff
@@ -0,0 +1,28 @@
+package datadog
+
+import (
+	"go.opentelemetry.io/otel/api/metric"
+	export "go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/array"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/lastvalue"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/sum"
+)
+
+type (
+	selectorDataDog struct{}
+)
+
+func NewWithDataDogMeasure() export.AggregationSelector {
+	return selectorDataDog{}
+}
+
+func (selectorDataDog) AggregatorFor(descriptor *metric.Descriptor) export.Aggregator {
+	switch descriptor.MetricKind() {
+	case metric.ObserverKind:
+		return lastvalue.New()
+	case metric.MeasureKind:
+		return array.New()
```

Comment:
What if a user wnted to use `ddsketch` instead of `array`?

I think we should have `NewWithDataDogMeasure` be more of a wrapper in that it should take an `exporter.Aggregator` and only override `metric.ObserverKind` -- something like this: 

```golang
type selectorDataDog struct {
  agg export.Aggregator
}

func NewWithDataDogMeasure(agg export.Aggregator) export.AggregationSelector {
	return selectorDataDog{}
}

func (s *selectorDataDog) AggregatorFor(descriptor *metric.Descriptor) export.Aggregator {
	switch descriptor.MetricKind() {
	case metric.ObserverKind:
		return lastvalue.New()
	default:
		return s.agg.AggregatorFor(descriptor)
	}
}
```

Reply:
I’ll leave this open for now. It’s a good idea, I just still need to do a bunch of testing with dd sketch and distribution metric type before I’m comfortable with exposing them to our user base.